cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -977,4 +1039,30 @@ private boolean isLiveNode(CassandraNode node) {
             return false;
         return info.getNativeTransportRunning() && info.getRpcServerRunning();
     }
+
+    public CassandraNode replaceNode(String node) throws ReplaceNodePreconditionFailed {
```

Comment:
Code coverage info can be seen here: https://teamcity.mesosphere.io/viewLog.html?buildId=12455&tab=coverage_idea&buildTypeId=Oss_Mesos_Cassandra_CassandraFramework